sentry/src/event_aggregator.rs

```diff
@@ -143,6 +143,12 @@ impl EventAggregator {
             _ => ResponseError::BadRequest(e.to_string()),
         })?;
 
+        events.iter()
+            .for_each(async |e| match e {
+                Event::UpdateTargeting { targeting_rules } => update_targeting_rules(&app.pool, &channel_id, &targeting_rules).await?,
+                _ => false,
```

Comment:
Should we use the `bool`? I think we should handle if it was successfully updated in the DB or not